webhook_to_fedora_messaging/config/defaults.py

```diff
@@ -0,0 +1,70 @@
+# SPDX-FileCopyrightText: Contributors to the Fedora Project
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+
+class Defaults:
+    DEBUG = True
+    TESTING = False
+    PROPAGATE_EXCEPTIONS = None
+    SECRET_KEY = "10505aec23775f6ecb61ded347654282d1be2c673e3b37b0372db6557f39f0ca"
```

Comment:
There shouldn't be a default `SECRET_KEY`. Flask alerts if none has been configured, it's safer use that than to risk someone not redefining it and using the default that everyone can see.